volume-providers/vpc/vpcclient/instances/iks_detach_volume.go

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018, 2019 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package instances
+
+import (
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/client"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/models"
+	"go.uber.org/zap"
+	"net/http"
+	"time"
+)
+
+// DetachVolume retrives the volume attach status with givne volume attachment details
+func (vs *IKSVolumeAttachService) DetachVolume(volumeAttachmentTemplate *models.VolumeAttachment, ctxLogger *zap.Logger) (*http.Response, error) {
+	defer util.TimeTracker("IKS DetachVolume", time.Now())
+
+	operation := &client.Operation{
+		Name:   "DetachVolume",
+		Method: "DELETE",
+	}
+	operation.PathPattern = vs.pathPrefix + "deleteAttachment"
+
+	apiErr := vs.receiverError
+
+	request := vs.client.NewRequest(operation)
+	request = request.SetQueryValue(IksClusterQueryKey, *volumeAttachmentTemplate.ClusterID)
+	request = request.SetQueryValue(IksWorkerQueryKey, *volumeAttachmentTemplate.InstanceID)
+	request = request.SetQueryValue(IksVolumeAttachmentIDQueryKey, volumeAttachmentTemplate.ID)
```

Comment:
Can you write a common method to  SetQueryValue ?  Its used from multiple functions